src/pydap/tests/test_model.py

```diff
@@ -299,76 +301,83 @@ def test_id(self):
         self.assertEqual(child.id, "child")
 
 
-class TestSequenceType(unittest.TestCase):
+@pytest.fixture
```

Comment:
I have started switching the tests to ``pytest`` when useful. In this case, I was only able to get the ``DeprecationWarning`` test to succeed with this approach.